pset-iterators.js

```diff
@@ -106,8 +117,18 @@ between 1850 and 1900. Save the data in a variable called `presidentsBetween1850
 ************/
 console.log('Problem 3:')
 
+// return all of the presidents then
+
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(dateInOffice => {
+   let stepUp = dateInOffice.took_office.slice(0,4)
+   if (stepUp < 1900 && stepUp > 1850) {
```

Comment:
@Defy1x nice because `.filter()` returns the element for every iteration that returns true you could have written this line this way (without the needed for the `if`)

`return stepUp < 1900 && stepUp > 1850`

Reply:
Noted.